### PR TITLE
[MIG][17.0] edi_exchange_template_oca: Migration to 17.0

### PR DESCRIPTION
**Pull request by john-herholz-dt**

### CODE REVIEW

**Comment by peluko00** [comment]
Can you push again @john-herholz-dt please

**Comment by john-herholz-dt** [comment]
I did but there is still an unmet dependency

**Comment by peluko00** [comment]
> I did but there is still an unmet dependency

It's normal because depends of other pr. Thanks!

**Review by peluko00 — APPROVED** [review]
LGTM, code review

**Comment by peluko00** [comment]
Test are failing @john-herholz-dt

File: edi_exchange_template_oca/tests/test_edi_backend_output.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        self.backend.exchange_generate(self.record3)
+        file_content = self.record3._get_file_content()
+        self.assertEqual(
+            self.report._render(self.report, [self.record3.res_id])[0]
+            .strip()
+            .decode("UTF-8"),
+            file_content.strip(),
+        )
+
+    def test_get_info_provider(self):
```
Comment (manuelregidor) [inline edi_exchange_template_oca/tests/test_edi_backend_output.py]: This test makes no checks.

**Review by manuelregidor — CHANGES_REQUESTED** [review]

**Comment by john-herholz-dt** [comment]
I removed the wrong test and added two tiny tests for the date functions to get more coverage.
Some coverage is missing but I think testing about a placeholder function like "validate: pass" does not make much sense.

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Review by HaraldPanten — APPROVED** [review]
@etobella WDYT? This PR is the other missing dependency for l10n_es_facturae_face.

THX!

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**